tests/unittests/MLTest.cpp

```diff
@@ -908,7 +908,7 @@ TEST_P(InterpreterAndCPU, convNetForImageRecognition) {
   auto *TANH = F->createTanh("tanh", CV);
   auto *FCL = F->createFullyConnected("fc", TANH, 2);
   auto *SM = F->createSoftMax("sm", FCL, ex);
-  auto *result = F->createSave("ret", SM);
+  auto *resultVar = F->createSave("ret", SM)->getVariable();
```

Comment:
why do we have all these changes? is it anyhow related to the bug_fix?

Reply:
because after erasing the old function, the save nodes that belong to original function also got removed. We need to a way to still be able to reference the variable of the save node